test/deployment/scaling/basic_scaling_deployment_test.go

```diff
@@ -50,7 +50,7 @@ var _ = ginkgo.Describe("DeploymentScalingBroker", func() {
 	})
 
 	ginkgo.It("Deploy single broker instance and scale up to max (16)", func() {
-		gomega.Expect(brokerDeployer.WithWait(false).DeployBrokers(1)).To(gomega.BeNil(), "Broker deployment failed")
+		gomega.Expect(brokerDeployer.WithWait(true).DeployBrokers(1)).To(gomega.BeNil(), "Broker deployment failed")
```

Comment:
This one is to fix a nasty problem plaguing scaling deployments creating random failures. Was a local change for awhile.